src/cephadm/node-proxy/redfish_system.py

```diff
@@ -80,12 +91,18 @@ def stop_update_loop(self):
     def update(self):
         #  this loop can have:
         #  - caching logic
-        while self.run:
-            self._update_system()
-            # following calls in theory can be done in parallel
-            self._update_metadata()
-            self._update_memory()
-            self._update_power()
-            self._update_network()
-            self._update_storage()
-            sleep(3)
+        try:
+            while self.run:
+                self._update_system()
+                # following calls in theory can be done in parallel
+                self._update_metadata()
+                self._update_memory()
+                self._update_power()
+                self._update_network()
+                self._update_storage()
+                sleep(3)
+        # Catching 'Exception' is probably not a good idea (devel only)
+        except Exception:
```

Comment:
even if this is just a devel thing, can we log what the exception actually was?

Reply:
sure